flytekit/core/array_node_map_task.py

```diff
@@ -113,8 +113,13 @@ def python_interface(self):
 
     def construct_node_metadata(self) -> NodeMetadata:
         # TODO: add support for other Flyte entities
+        interruptible = None
+        if self._run_task.metadata:
+            interruptible = self._run_task.metadata.interruptible
+
         return NodeMetadata(
             name=self.name,
+            interruptible=interruptible,
```

Comment:
Should we also copy other values from task metadata?

Reply:
This is all kinds of weird. So I added the cache metadata to NodeMetadata to support subworkflows / launchplans. IMO caching should have been set at the Node level, but for task executions its embedded in the TaskTemplate, so not necessary here. 

cc @pvditt can you expand on your other PR (since closed) to update the timeouts here.